src/azure-cli/azure/cli/command_modules/appservice/_create_util.py

```diff
@@ -44,6 +44,15 @@ def zip_contents_from_dir(dirPath, lang):
                 subdirs[:] = [d for d in subdirs if d not in ['obj', 'bin']]
             elif lang.lower() == PYTHON_RUNTIME_NAME:
                 subdirs[:] = [d for d in subdirs if 'env' not in d]  # Ignores dir that contain env
+
+            filtered_files = []
+            for filename in files:
+                if filename == '.env':
```

Comment:
also should we scope this to Python only? and run this check every time?

Reply:
Scoped to python only